### ISSUE TITLE
support reoccurring payments / subscriptions

### ISSUE THREAD

**Issue by p0x6** [brendanbond/react-acceptjs#2]

**Comment by brendanbond** [comment]
Yeah, the Accept Hosted portion of this library could support [getting a customer profile form key](https://developer.authorize.net/api/reference/index.html#accept-suite-get-accept-customer-profile-page). I'll put that at the top of the list! I might even split the Accept Hosted portion off into a separate library because it's so different. Is this the component you're using?

If you're using either the `useAcceptJs()` hook or the `<HookForm />` component, though, this would be done server-side; you can use the payment nonce (`opaqueData`) that either returns to set up subscriptions and/or customer profiles.
